GooglePlayInstant/Editor/QuickDeploy/AssetBundleVerifierWindow.cs

```diff
@@ -25,32 +29,44 @@ public class AssetBundleVerifierWindow : EditorWindow
     {
         private const int FieldMinWidth = 170;
 
-        private bool _assetBundleDownloadIsSuccessful;
-        private string _assetBundleUrl;
+        internal bool AssetBundleDownloadIsSuccessful;
```

Comment:
Group the internal fields together and add comment

// Visible for testing

Reply:
Resolved.